cupboard_app/test_queries.py

```diff
@@ -166,6 +167,18 @@ def test_get_custom_ingredient(self):
         with self.assertRaises(CustomIngredient.DoesNotExist):
             get_custom_ingredient(username=self.user.username, name='doesnt_exist')
 
+    def test_delete_custom_ingredient(self):
+        """
+        Testing delete_custom_ingredient deletes a custom ingredient from the database
+        """
+        self.assertEqual(len(get_all_custom_ingredients(self.user.username)), 2)
+        delete_custom_ingredient(self.user.username, self.ing1.name)
+        self.assertEqual(len(get_all_custom_ingredients(self.user.username)), 1)
+        delete_custom_ingredient(self.user.username, 'does not exist')
+        self.assertEqual(len(get_all_custom_ingredients(self.user.username)), 1)
+        with self.assertRaises(User.DoesNotExist):
+            delete_custom_ingredient('does not exist', self.ing1.name)
+
```

Comment:
Instead of using `get_all_custom_ingredients` function here, use the django queries directly so that this test doesn't rely on the function to succeed.

Also add a test to try deleting when there are no custom ingredients. Maybe take a look at the `test_delete_user_list_ingredients` test function in the `UserListIngredientsQueries` class as an example. Don't need to follow that function because it's not the best, but just as a reference.